### ISSUE TITLE
Branch list sort order broken?

### ISSUE THREAD

**Issue by Turbo87** [ForkIssues/Tracker#272]
I've noticed lately that the sort order for the branch list in the sidebar seems to be somewhat broken:

<img width="188" alt="bildschirmfoto 2018-04-27 um 17 59 29" src="https://user-images.githubusercontent.com/141300/39372526-ee65498a-4a44-11e8-8d76-c2a8879269d1.png">

I'm not sure what the exact pattern is, but I've noticed that branches beginning with "a" or "b" seems to be sorted to the wrong position in the list 🤔

**Comment by Turbo87** [comment]
```
57b65fd10fda2880e7a216c2f660b3fde4f9aa69 refs/heads/base-map-cleanup
6338ed2d35df2fe2b6f60c5d0ca5ab3ccd306ef8 refs/heads/bundlesize
4fc1665b6c6cbba092258611f0bcd2f17d4f1f9c refs/heads/elevation-display
8abea49533b6ef41fa03d28681dc73dc8baa3a32 refs/heads/ember-font-awesome
80fd54eeea143202b0744f8f51150cea752d9254 refs/heads/ember-string
744a7896fe4b5b55d9b07cad9ee1a2f2b3d48caf refs/heads/favicon
5ec450373708a7bd5a16d5f77ba9f46fb3a23b20 refs/heads/geoip
c8c6d9649e056aaf4dca455b4faa4c862d1c0c23 refs/heads/jsonapi
0e967693cd30f2acc2cedf426c4adff46d7fad80 refs/heads/last-fix
3423118c1c44cf1f219194f7a7d6f1f09d4100d7 refs/heads/live-api
8fe5ee0c6933ce29ac3b652b9c18cc2731d5b22f refs/heads/master
b12068bc0ec5929cb002cd2ff9e87fcd66093679 refs/heads/ogn
fc89ba1fde511a9bc715dd05a3c913c20126d6ac refs/heads/ogn-api-proxy
5108768f4a27177599627be5db3a715910f285d6 refs/heads/pipenv
```

looks regular to me 🤔

**Comment by DanPristupov** [comment]
Yes. BTW, same names give me correct result.
I checked the code, it's really straightforward. I'm very curious how this can happen.

**Comment by Turbo87** [comment]
let me know if you want me to run any other git commands to get more info 😉

**Comment by DanPristupov** [comment]
I know, it's late, but I've fixed that in 1.0.72 :)
